### PR TITLE
Vendoring openshift

### PR DESCRIPTION
**Pull request by surajnarwade**
What is the purpose of this change? What does it change?

This PR,
* Added openshift as a vendor
* kubernetes projects will point to openshift's forked repos

How it helps ?

* This will help me in `odo login` PR
* since, we are dealing with openshift only, it's good to point to kubernetes repos which are maintained by openshift (because they make changes and maintain in their own org)

### CODE REVIEW

**Comment by kadel** [comment]
> What is the purpose of this change? What does it change?
> 
> Was the change discussed in an issue?
> 
> How to test changes?

no description :-(

File: .wwhrd.yml [hunk 1]
```diff
@@ -16,3 +16,9 @@ exceptions:
   - github.com/davecgh/go-spew/spew/...
   # wwhrd currently does not detect ISC which is compatible with Odo so, add it under exceptions to avoid errors due to it being not recognised
   - github.com/olekukonko/tablewriter
+  # gocertifi contains CA bundle and SSL certs which is required for OpenShift
+  # This is OpenShift dependency, ODO do not use this directly
+  - github.com/certifi/gocertifi
```
Comment (kadel) [inline .wwhrd.yml]: what licence is it?

File: .wwhrd.yml [hunk 2]
```diff
@@ -16,3 +16,9 @@ exceptions:
...
+  # go-digest contains common digest package used across the container ecosystem
+  # This is OpenShift dependency, ODO do not use this directly
+  - github.com/opencontainers/go-digest
```
Comment (kadel) [inline .wwhrd.yml]: what license is it?

File: .wwhrd.yml (hunk 1 above)
Comment (surajnarwade) [reply .wwhrd.yml]: removed since it has MPL 2.0 license

File: .wwhrd.yml (hunk 2 above)
Comment (surajnarwade) [reply .wwhrd.yml]: removed since it has apache license

**Comment by metacosm** [comment]
What's the point of that change?

**Comment by surajnarwade** [comment]
@metacosm , @kadel , I updated the description :smile:

File: .wwhrd.yml [hunk 3]
```diff
@@ -15,4 +15,4 @@ exceptions:
   # wwhrd currently does not detect ISC which is compatible with Odo so, add it under exceptions to avoid errors due to it being not recognised
   - github.com/davecgh/go-spew/spew/...
   # wwhrd currently does not detect ISC which is compatible with Odo so, add it under exceptions to avoid errors due to it being not recognised
-  - github.com/olekukonko/tablewriter
+  - github.com/olekukonko/tablewriter
```
Comment (kadel) [inline .wwhrd.yml]: ?

File: glide.yaml [hunk 4]
```diff
@@ -40,4 +43,9 @@ import:
   version: d4647c9c7a84d847478d890b816b7d8b62b0b279
 - package: github.com/posener/complete
   version: v1.1.2
+- package: github.com/openshift/origin
```
Comment (kadel) [inline glide.yaml]: shouldn't `github.com/openshift/origin` and `github.com/openshift/client-go` versions match?

File: glide.yaml [hunk 5]
```diff
@@ -40,4 +43,9 @@ import:
...
+  version: v3.11.0
+- package: k8s.io/kubernetes
```
Comment (kadel) [inline glide.yaml]: why is this here?  Glide should be smart and detect shi from openshift import.

File: glide.yaml (hunk 4 above)
Comment (kadel) [reply glide.yaml]: we are using just one package from origin, why not just import that subpackage?

Comment (surajnarwade) [reply glide.yaml]: @kadel , https://github.com/openshift/client-go/releases don't release regularly, so I took it from the master then :(

Comment (surajnarwade) [reply glide.yaml]: openshift directly uses `master` from `openshift/client-go`
ref: https://github.com/openshift/origin/blob/master/glide.yaml#L42
shall I use the same ?

Comment (surajnarwade) [reply glide.yaml]: I created issue for the same: https://github.com/openshift/client-go/issues/66

Comment (kadel) [reply glide.yaml]: only origin master uses client-go master. If you check glide.yaml for 3.11 you can see that they are using client-go 3.11 https://github.com/openshift/origin/blob/v3.11.0/glide.yaml#L43

**Comment by surajnarwade** [comment]
@kadel updated

**Comment by cdrage** [comment]
@surajnarwade You need to update the commit message too :+1:

**Comment by surajnarwade** [comment]
@cdrage updated with commit message

**Comment by surajnarwade** [comment]
@kadel , can you please review once ?
